devtools-ci-index.yaml

```diff
@@ -2285,14 +2285,14 @@
             timeout: '20m'
         - '{ci_project}-{git_repo}-fabric8-analytics':
             git_organization: fabric8-analytics
-            git_repo: 3scale-admin-gateway
+            git_repo: f8a-3scale-connect-api
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_run_tests.sh'
             timeout: '20m'
-            image_name: 'fabric8-analytics/3scale-admin-gateway'
+            image_name: 'fabric8-analytics/f8a-3scale-connect-api'
```

Comment:
Makefile still references the old image name: https://github.com/fabric8-analytics/f8a-3scale-connect-api/blob/master/Makefile#L2

Reply:
@msrb, I have fixed it here. https://github.com/fabric8-analytics/f8a-3scale-connect-api/commit/f7e53aa177ceff875285c9274923f130508657c0